lib/simp/beaker_helpers/constants.rb

```diff
@@ -0,0 +1,24 @@
+module Simp; end
+
+module Simp::BeakerHelpers
+  # This is the *oldest* version that the latest release of SIMP supports
+  #
+  # This is done so that we know if some new thing that we're using breaks the
+  # oldest system that we support
+  DEFAULT_PUPPET_AGENT_VERSION = '1.10.4'
+
+  if ['true','yes'].include?(ENV['BEAKER_online'])
+    ONLINE = true
+  elsif ['false','no'].include?(ENV['BEAKER_online'])
+    ONLINE = false
+  else
+    require 'open-uri'
+
+    begin
+      ONLINE = true if open('http://google.com')
```

Comment:
We don't want to do this. We need to hit something that lets us know that we're actually on the Internet since LOTS of things are going to have to change if we aren't.

That said, it makes sense to make it an environment variable in case people are behind weird proxies or something.